src/components/MobileNavbar/MobileNavbarAction.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import cx from 'classnames'
+import { Button, Icon } from '@santiment-network/ui'
+import styles from './MobileNavbarAction.module.scss'
+
+const MobileNavbarAction = ({
+  onClick,
+  iconType,
+  label,
+  linkTo,
+  classes = {
+    isActive: styles.isActive,
+    icon: styles.icon,
+    wrapper: styles.wrapper,
+    label: styles.label,
+    button: styles.button
+  },
+  isActive = false,
+  className = ''
+}) => {
+  const handleOnClick = () => onClick(linkTo)
+
+  return (
+    <Button
+      className={cx(
+        { [classes.isActive]: isActive },
+        classes.button,
+        className
+      )}
+      isActive={isActive}
+      onClick={handleOnClick}
+    >
+      <Icon
+        type={iconType}
+        className={classes.icon}
+        fill={isActive ? 'var(--jungle-green)' : '#000'}
+      />
+      <div className={cx(classes.label, { [classes.isActive]: isActive })}>
```

Comment:
`cx(classes.label, { [classes.isActive]: isActive })` could be much simplier, without passing an extra object, since there are no more conditional classes:`cx(classes.label, isActive && classes.isActive })`